code/modules/mob/dead/new_player/sprite_accessories.dm

```diff
@@ -2454,6 +2454,16 @@
 	name = "Zeng-Hu Pharmaceuticals"
 	limbs_id = "zhpipc"
 
+/datum/sprite_accessory/ipc_brain
+	icon = null
+	icon_state = "not important"
+
+/datum/sprite_accessory/ipc_brain/posibrain
+	name = "Posibrain"
+
+/datum/sprite_accessory/ipc_brain/mmi
+	name = "Man-Machine Interface"
+
```

Comment:
I beg you, please just manually make the global list, these have zero reason to exist